### DISCUSSION TITLE
Maps look different in Google Colab

### DISCUSSION THREAD

**Discussion by shi093** [opengeos/leafmap#82] [Q&A] [answered]
The map rendered in Google Colab looks different from the same map rendered on my local computer:
1. The information box "i" on the right hand side does not show up in Google Colab. Is there someway to display the data field information for the maps rendered in Google Colab in addition to just lon and lat when I clicked the polygon?
2. The "toolbox" is not available in Google Colab
3. The layer style looks different in Google Colab. The polygon boundary color is specified to be "white", but it shows in the fillColor
Why my map looks different in Google Colab? Thanks a lot.

**Comment by shi093** [comment]
I figured out the item #3, I need to repeat the parameters specified in style to style_callback, as I used the style_callback to specify the fill color based on one data field. This works differently on local computer and Colab, but it works anyway.
I really need to figure out the "i" box thought. Please help. Thanks again.

**Comment by giswqs** [comment, marked as answer]
Google Colab does not ipyleaflet. See https://leafmap.org/faq/#why-does-leafmap-use-two-plotting-backends-folium-and-ipyleaflet
Try binder: https://gishub.org/leafmap-pangeo
